src/internal/util/errors.ts

```diff
@@ -0,0 +1,68 @@
+
+/**
+ * Used to identify different types of RxJS errors.
+ */
+export const enum RxErrorCode {
+  Empty = 0,
+  OutOfRange = 1,
+  ObjectUnsubscribed = 2,
+  Timeout = 3,
+  Teardown = 4,
+}
```

Comment:
We cannot use const enums. It will break with isolated modules. They cannot even be exported within internal modules, IIRC.

Reply:
IMO, we should get this - or something very much like it - merged: https://github.com/ReactiveX/rxjs/pull/4677